packages/api/utils/database.ts

```diff
@@ -80,7 +85,7 @@ export async function createConnection(): Promise<Connection> {
     },
     logging: "all",
     database: databaseName,
-    entities: ["/opt/nodejs/orm/entity/**.js"],
+    entities: [Portfolio, Application, Environment, TaskOrder, Clin, "/opt/nodejs/orm/entity/**.js"],
```

Comment:
Will we need to do this forever for all new entities or is this a stop-gap to get through the initial cut of implementations?

Reply:
This is a documented capability.  Reference [this comment](https://github.com/dod-ccpo/atat-web-api/pull/518#discussion_r767985259) where I addressed similar question on #518

Reply:
I think we also need the *.js references when running migrations.

Reply:
I'm not completely sure. @jasonburkert-ccpo found this work around and might have a better idea as if this is needed or if there is a better way.

Reply:
Got it. No big deal for now but we can get to the bottom of it later. Not ideal to have to remember to put new entities in a config file but also not the end of the world.